backend/deepchecks_monitoring/schema_models/alert_webhook.py

```diff
@@ -307,7 +307,7 @@ class PagerDutyWebhookProperties(BaseModel):
     name: str
     description: str
     notification_levels: t.Optional[t.List[AlertSeverity]] = None
-    api_access_key: str
+    api_access_key: t.Optional[str] = None
```

Comment:
additionally, you need to modify `PagerDutyWebhookProperties.access_token` and `PagerDutyWebhookProperties.as_values`

also, till you have a pager duty trial access, try to run `test_pager_duty_webhook_execution` from `tests.unittest.test_pager_duty_webhook_execution` (this test does not run without two required env vars)

Reply:
what should be changed in as_values? i already modified PagerDutyWebhookProperties.access_token, this is the line you commented on

Reply:
for as_values, do you mean not to add acess_token if its None?

Reply:
**Point 1.**

leave `http_headers` empty if `api_access_key is None`

```
def as_values(self) -> t.Dict[str, t.Any]:
    ...
    "http_headers": {"Authorization": self.access_token} if self.api_access_key else {},
    ...
```

**Point 2.**

I mean this property

```
    @property
    def access_token(self) -> str:
        """Return PagerDuty 'Authorization' header value."""
        return f"Token token={self.api_access_key}" if self.api_access_key else ""
```

return an empty string if `api_access_key is None`

**Point 3.**

Great)